minions.md

```diff
@@ -300,7 +320,7 @@ You can train Firemaking using `+light [quantity] <logs>`, for example `+light 5
 
 Click [here](https://i.imgur.com/80iIwN9.png) for **Firemaking** XP rates.
 
-You can now also train Firemaking via the Wintertodt minigame using the `+wt` or `+wintertodt` commands. Fighting the Wintertodt provides item rewards scales to your Woodcutting, Mining, Fishing, Farming, Crafting, and Herblore levels, and xp scaled to your Firemaking level. You need level 50 Firemaking to challenge the Wintertodt, and each attempt requires food to heal yourself from the Wintertodt's attacks, currently a fish which heals >10hp. The speed at which you challenge the Wintertodt increases with a higher Woodcutting level, and the food required is reduced for every piece of the pyromancer outfit you have equipped in your skilling setup, as well as the warm gloves and the fire cape. 
+You can now also train Firemaking via the Wintertodt minigame using the `+wt` or `+wintertodt` commands. Fighting the Wintertodt provides item rewards scales to your Woodcutting, Mining, Fishing, Farming, Crafting, and Herblore levels, and xp scaled to your Firemaking level. You need level 50 Firemaking to challenge the Wintertodt, and each attempt requires food to heal yourself from the Wintertodt's attacks, currently a fish which heals >10hp. Your worst food (the food which heals the least) will be used first before better (and more expensive food) is used. The speed at which you challenge the Wintertodt increases with a higher Woodcutting level, and the food required is reduced for every piece of the pyromancer outfit you have equipped in your skilling setup, as well other items of [warm clothing](https://oldschool.runescape.wiki/w/Wintertodt/Warm_clothing).
```

Comment:
the `currently a fish which heals >10hp` isnt applicable anymore.  I dont think `Your worst food (the food which heals the least) will be used first before better (and more expensive food) is used.` is really necessary, but if you wanted to keep it, perhaps say something more like "Lower quality food is used first"